src/pki/crypto.ts

```diff
@@ -0,0 +1,31 @@
+import { webcrypto } from "crypto";
+import { getCrypto, setEngine } from "pkijs/src/common";
+import CryptoEngine from "pkijs/src/CryptoEngine";
+
+export const initCrypto = (): CryptoEngine => {
+    let cryptoEngine: CryptoEngine = getCrypto() as CryptoEngine;
```

Comment:
The code can be executed in a node context or in a browser context which need to be handled differently. `cryptoEngine` can be undefined when the WebCrypto API is not available in a specific browser or node environment (e.g. node < 16). In case it is available, it needs to be wrapped by pkijs `setEngine` for pkijs cryptographic methods to work.